frontend/src/app/components/containers/complaints/outcomes/hwcr-prevention-education.tsx

```diff
@@ -288,6 +288,7 @@ export const HWCRComplaintPrevention: FC = () => {
                         className="comp-details-edit-calendar-input" // Adjust class as needed
                         classNamePrefix="comp-select" // Adjust class as needed
                         errMsg={preventionDateErrorMessage} // Pass error message if any
+                        maxDate={new Date()}
```

Comment:
If you want to pass a date down to the child components like this, you will want to create a top level ```const currentDate: Date = new Date()``` and pass it down instead of a new date, this way the date is consistent for everything in the component.

Reply:
Makes sense